sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/CreateDigitalTwinOptions.cs

```diff
@@ -18,5 +18,15 @@ public partial class CreateDigitalTwinOptions
         /// <summary> Provides vendor-specific trace identification information and is a companion to TraceParent. </summary>
         [CodeGenMember("Tracestate")]
         public string TraceState { get; set; }
+
+        /// <summary>
+        /// If-Non-Match header that makes the request method conditional on a recipient cache or origin server either not having any current representation of the target resource.
+        /// For more information about this property, see <see href="https://tools.ietf.org/html/rfc7232#section-3.2">RFC</see>.
+        /// Acceptable values are null or "*".
+        /// If IfNonMatch option is null the service will replace the existing entity with the new entity.
+        /// If IfNonMatch option is "*" the service will reject the request if the entity already exists.
+        /// </summary>
+        [CodeGenMember("IfNoneMatch")]
+        public string IfNoneMatch { get; set; }
```

Comment:
This is a well known header value for advanced users, I did not rename it , what does everyone think ?

Reply:
Seems fine to me